### DISCUSSION TITLE
v2.31.0 doesn't include subtitle

### DISCUSSION THREAD

**Discussion by zash13x** [staxrip/staxrip#1172] [Q&A]
I downloaded and updated my StaxRip folder to the v2.31.0 and tried encoding videos and when I checked the output files, the srt files were not included.

I am using the same settings that I have in v2.30.0 and tried encoding the same files using it with the same settings, it works just fine, subtitles got included in the output.

Can anyone help?

**Comment by Dendraspis** [comment]
The solution is to use fresh/clean/new settings and templates. There were some changes to the language handling that interfere with your old settings/templates.

**Comment by zash13x** [comment]
> The solution is to use fresh/clean/new settings and templates. There were some changes to the language handling that interfere with your old settings/templates.

@Dendraspis 

I tried what you suggested. Fresh settings for v2.31.0.
From my experience, Options > Subtitle settings behaves differently compared to the previous version.

Edit/Update:
I tried an MKV file with 2 subtitle files inside it.
What v2.31.0 does is that it does not include the 1st subtitle. It only includes the 2nd.
I tried different subtitle settings and the result is still the same.
So if the MKV only has 1 subtitle file, it is automatically not included.

Can someone please explain as to why this version behaves like this?

**Reply by Dendraspis** [reply]
On `Options » Subtitles`, the first option is called `Subtitles`, which value does it have?
And I definitely need more info about the source file.

Also when you're sure there is a problem, please open an issue and provide more details how to reproduce it as this is just a discussion...

**Comment by zash13x** [comment]
> On `Options » Subtitles`, the first option is called `Subtitles`, which value does it have? And I definitely need more info about the source file.
> 
> Also when you're sure there is a problem, please open an issue and provide more details how to reproduce it as this is just a discussion...

Hi! I have tried all the settings available there and it always fail to include the 1st subtitle file.

> I tried different subtitle settings and the result is still the same.
> So if the MKV only has 1 subtitle file, it is automatically not included.

**Reply by zash13x** [reply]
@Dendraspis , if it is convenient for you, would you mind providing me with a video file in MKV format that includes 2 subtitle tracks? My intention is to conduct a screen recording of my StaxRip settings and share the results with you.

Thank you in advance for considering my request and assisting me in further deep diving into what is really happening.

**Reply by Dendraspis** [reply]
> would you mind providing me with a video file in MKV format that includes 2 subtitle tracks?

I don't get it; why do you want a video file from me? I would make more sense to provide me a video file so I can reproduce your issues.

According to ##1174 it maybe better to wait for the next release and see, if your issues are already solved with the next release. There will be some bigger changes. 
Otherwise open an issue/bug-report and provide proper information to reproduce your issue as well as answer the questions I have already asked.
